src/evaluate.py

```diff
@@ -36,7 +36,7 @@ def load_artifacts(test_set_file, trained_model_file):
     print("Loading language model...")
     nlp = spacy.load("en_core_web_md")
     print("Loading machine-learning model...")
-    model = joblib.load(trained_model_file)
+    model = cloudpickle.load(trained_model_file)
```

Comment:
the file name could also change to a `pkl` extension maybe?

Reply:
yup we can do that